scripts/e2e-local.sh

```
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+# Local e2e runner: sources .env then dispatches a single spec/preset.
+# Usage:
+#   scripts/e2e-local.sh regression                    # beautifului polish regression
+#   scripts/e2e-local.sh mobile                        # mobile-viewport e2e
+#   scripts/e2e-local.sh smoke                         # smoke spec
+#   scripts/e2e-local.sh audit [sidebar|fullscreen|mobile]  # design-audit screenshots
+#   scripts/e2e-local.sh test/e2e/specs/<file>.e2e.ts  # any spec
+set -euo pipefail
+ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
+cd "$ROOT"
+if [ -f .env ]; then set -a; source .env; set +a; fi
```

Shellcheck (0.11.0)

[info] Not following: .env was not specified as input (see shellcheck -x).

(SC1091)